bundle/manifests/k8s-nim-operator.clusterserviceversion.yaml

```diff
@@ -146,6 +146,27 @@ metadata:
             ]
           }
         },
+        {
+          "apiVersion": "apps.nvidia.com/v1alpha1",
+          "kind": "NIMBuild",
+          "metadata": {
+            "name": "nim-build-sample",
+            "namespace": "nim-service"
+          },
+          "spec": {
+            "nimCache": {
+              "name": "meta-llama3-2-3b-instruct",
```

Comment:
it will be good to match this to the cache example we have `meta-llama3-8b-instruct`.